portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -714,375 +836,441 @@ definition {
+				WebContentDisplayPortlet.selectWebContent(webContentTitle = "Web Content Title ${count}");
+			}
 		}
 
-		LAR.exportSiteCP(siteScopeName = "Site A");
+		task ("When: User exports the site and creates a new site with 3 pages") {
+			LAR.exportSiteCP(siteScopeName = "Site A");
 
-		var key_larFileName = "${larFileName}";
-		var larFileName = LAR.getLarFileName();
+			var key_larFileName = "${larFileName}";
+			var larFileName = LAR.getLarFileName();
 
-		LAR.downloadLar();
+			LAR.downloadLar();
 
-		JSONGroup.addGroup(groupName = "Site B");
+			JSONGroup.addGroup(groupName = "Site B");
 
-		for (var count : list "A,B,C") {
-			JSONLayout.addPublicLayout(
-				groupName = "Site B",
-				layoutName = "Widget ${count}");
+			for (var count : list "A,B,C") {
+				JSONLayout.addPublicLayout(
+					groupName = "Site B",
+					layoutName = "Widget ${count}");
+			}
 		}
 
-		JSONGroup.deleteGroupByName(groupName = "Site A");
-
-		LAR.importSiteCP(
-			larFileName = "${larFileName}",
-			siteName = "Site B");
+		task ("And: Deletes the first site then import the LAR file on the new site") {
+			JSONGroup.deleteGroupByName(groupName = "Site A");
 
-		Navigator.gotoSitePage(
-			pageName = "Page A",
-			siteName = "Site B");
-
-		WebContentDisplayPortlet.viewContent(
-			displayText = "Link to Page",
-			pageName = "Page B",
-			webContentTitle = "Web Content Title A");
+			LAR.importSiteCP(
+				larFileName = "${larFileName}",
+				siteName = "Site B");
+		}
 
-		WebContentDisplayPortlet.viewContent(
-			displayText = "Link to Page",
-			pageName = "Page C",
-			webContentTitle = "Web Content Title B");
+		task ("Then: The WCD portlets will be present with the links to the other pages") {
+			Navigator.gotoSitePage(
+				pageName = "Page A",
+				siteName = "Site B");
+
+			WebContentDisplayPortlet.viewContent(
+				displayText = "Link to Page",
+				pageName = "Page B",
+				webContentTitle = "Web Content Title A");
+
+			WebContentDisplayPortlet.viewContent(
+				displayText = "Link to Page",
+				pageName = "Page C",
+				webContentTitle = "Web Content Title B");
+		}
 	}
 
 	@description = "This is a use case for LPS-88378. Verify is possible export/import layout circular reference."
 	@priority = "3"
 	test ExportImportLayoutCircularReference {
 		property testray.component.names = "Site Pages Administration,Web Content Administration";
 
-		JSONGroup.addGroup(groupName = "Site Name");
+		task ("Given: User adds a new site and a page to it") {
+			JSONGroup.addGroup(groupName = "Site Name");
 
-		JSONLayout.addPublicLayout(
-			groupName = "Site Name",
-			layoutName = "Site Page");
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Site Page");
+		}
 
-		JSONLayout.addPublicLayout(
-			groupName = "Site Name",
-			layoutName = "Site Child Page",
-			parentLayoutName = "Site Page");
+		task ("And: Add child page to the first page page then a child page to this page") {
```

Comment:
Should be "And: Add a child page to the first page then a child page to this page"